app/server/app.js

```diff
@@ -1,13 +1,37 @@
 import express from 'express';
+import passport from 'passport';
```

Comment:
The main file of application usually becomes complex, according to the number of features implemented. Usually, the file has the responsibility to connect and delegate the available features from different directories.

E.g.:

`app.js`: Invokes the main functions from routes, middlewares, swagger, and additional services/libraries. The modularity of express.js allows importing the external files/modules/packages just passing the `app` by argument:
```javascript
const db = require('my-database');
const express = require('express');
const app = express();

middlewares(app);
authentication(app, db);
routes(app);
```
`/routes/index.js`: Keeps the logic to initialize all routes, declaring them by order and only exposing one single function to initialize itself.
`/middlewares/index.js` or `middlewares.js`: Keeps the logic to initialize all middlewares, declaring them by order and only exposing one single function to initialize itself, like `bodyParser`, `parseCookies`, `parseQuery`.
`/authentication/index.js` or `authentication.js`: Allows to configure the passport or another authentication strategy/library separately, isolating the context from rest of application. If a database read is required, it can be done passing the instance of a database by argument or reading inside of module itself. Depends on the database.

This modularization structure allows the developer to extend the functionalities without major issues. If additional inputs are required by sub-modules, they can be declared on the main file and passed as an argument to the module main function.